.github/workflows/pull_request_title.yml

```diff
@@ -0,0 +1,21 @@
+name: Gitmoji + Commitlint Pull Request Title Rule 
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: gitmoji + commitlint title rule
+        run: |
+          npm install --save-dev @commitlint/cli commitlint-config-gitmoji
+          echo "module.exports = {extends: ['gitmoji']}" > commitlint.config.js
```

Comment:
Could you commit a `commitlint.config.js` file instead? Other configurations are stored there instead of creating them on a per-build basis. This'll also make it easier for folks to run the commands locally.

The `.gitignore` ignoring `.js` files is just for compiled outputs in project directories. You can add an exclude for root-level `.js` files if you want, or just another one for `!commitlint.config.js` akin to the existing `!jest.config.js`.